src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3907,30 +3907,53 @@ void ApplicationManagerImpl::SendDriverDistractionState(
     LOG4CXX_WARN(logger_, "DriverDistractionState is INVALID_ENUM");
     return;
   }
-  smart_objects::SmartObjectSPtr on_driver_distraction =
-      std::make_shared<smart_objects::SmartObject>();
 
-  (*on_driver_distraction)[strings::params][strings::message_type] =
-      static_cast<int32_t>(application_manager::MessageType::kNotification);
-  (*on_driver_distraction)[strings::params][strings::function_id] =
-      mobile_api::FunctionID::OnDriverDistractionID;
-  (*on_driver_distraction)[strings::msg_params][mobile_notification::state] =
-      driver_distraction_state();
-  (*on_driver_distraction)[strings::params][strings::connection_key] =
-      application->app_id();
+  auto create_notification = [application, this]() {
+    auto notification = std::make_shared<smart_objects::SmartObject>();
+    auto& msg_params = (*notification)[strings::msg_params];
+    auto& params = (*notification)[strings::params];
+
+    params[strings::message_type] =
+        static_cast<int32_t>(MessageType::kNotification);
+    params[strings::function_id] = static_cast<int32_t>(OnDriverDistractionID);
```

Comment:
I am a bit confused as to why this builds, since the `using` was removed. Seems it would need to be `mobile_apis::FunctionID::OnDriverDistractionID` here.

Reply:
@jacobkeeler Thanks. Added in 42d1dda4207c4d14e7ca659e5e34e716b448013e